index.html

```diff
@@ -65,54 +68,56 @@
-    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/PqHQeu6bPqQ" frameborder="0" allow="encrypted-media" allowfullscreen></iframe>
-  </div>
-
   <div class="askepott x">
-    <div class="askepott y">
-      <img class="askepott" src="./images/askepott.gif" width="600" height="462" alt="Askepott"/>
+      <div class="askepott y">
+        <img class="askepott" src="./images/askepott.gif" width="600" height="462" alt="Askepott"/>
+      </div>
     </div>
+  <div id="fridayholder"></div>
+
+  <div style="display: flex; width: 100%; justify-content: space-between;">
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/NVWlPhG7qx0?rel=0" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen></iframe>
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/PqHQeu6bPqQ" frameborder="0" allow="encrypted-media" allowfullscreen></iframe>
   </div>
 
   <div class="frame">
     <canvas id="tree"></canvas>
   </div>
-  <div class="frame">
-    <div id="candles" style="position: relative; right: 38px; top: 20px;">
-      <div class="candle">
+  <div id="candles">
+    <div class="candle">
+      <div class="flamegroup">
+        <div class="flame" id="fl1"></div>
+        <div class="flame" id="fl2"></div>
+        <div class="flame" id="fl3"></div>
+      </div>
+      </div>
+      <div class="candle noFlame">
         <div class="flamegroup">
           <div class="flame" id="fl1"></div>
           <div class="flame" id="fl2"></div>
           <div class="flame" id="fl3"></div>
         </div>
+      </div>
+      <div class="candle noFlame">
+        <div class="flamegroup">
+          <div class="flame" id="fl1"></div>
+          <div class="flame" id="fl2"></div>
+          <div class="flame" id="fl3"></div>
         </div>
-        <div class="candle">
-          <div class="flamegroup">
-            <div class="flame" id="fl1"></div>
-            <div class="flame" id="fl2"></div>
-            <div class="flame" id="fl3"></div>
-          </div>
-        </div>
-        <div class="candle">
-          <div class="flamegroup">
-            <div class="flame" id="fl2"></div>
-
-            <div class="flame" id="fl1"></div>
-            <div class="flame" id="fl3"></div>
-          </div>
-        </div>
-        <div class="candle noFlame">
-          <div class="flamegroup ">
-            <div class="flame" id="fl1"></div>
-            <div class="flame" id="fl2"></div>
-            <div class="flame" id="fl3"></div>
-          </div>
+      </div>
+      <div class="candle noFlame">
+        <div class="flamegroup ">
+          <div class="flame" id="fl1"></div>
+          <div class="flame" id="fl2"></div>
+          <div class="flame" id="fl3"></div>
         </div>
       </div>
-    </div>
+  </div>
   </body>
-  <script type="text/javascript" src="snowstorm.js"></script>
+  <!-- <script type="text/javascript" src="snowstorm.js"></script> -->
```

Comment:
Ingen snøstorm? ❄️ 🎅 
```suggestion
  <script type="text/javascript" src="snowstorm.js"></script>
```

Reply:
Omg nice med commit suggestion! :D 

Jeg måtte disable snøstorm 🌬 💨 ❄️❄️❄️  under utvikling, fordi det var så ekstremt vanskelig å inspecte dom'en med snøstorm på.